src/Hive.IO/Hive.IO/Results/ResultsPlotting.cs

```diff
@@ -191,7 +191,11 @@ public double[] OperationEnergySystemsMonthly(bool normalized)
         public float InternalGains => (float)Results.TotalInternalGains;
         public float PrimaryEnergy => (float)Results.TotalPrimaryEnergyMonthlyNonRenewable.Sum(); //inputCarriers from conversionTech, except renewable tech (solar). 
         public float RenewableEnergy => (float)Results.TotalFinalEnergyMonthlyRenewable.Sum(); // solar tech
-        
+        public float VentilationGains => (float) Results.TotalVentilationHeatLosses;    // only take the negative values from this list, and make them Math.Abs( )
+        public float EnvelopeGains => (float) Results.TotalOpaqueTransmissionHeatLosses; // same as above comment
```

Comment:
`// same as above comment` should be avoided, since who knows what the above comment will end up being? if you're going to type stuff anyway, just type it. Also... I don't understand the above comment (`// only take the negative values from this list, and make them Math.Abs(  )`) - it doesn't seem to be what you're doing at all?

Reply:
wait... is that supposed to be a hint to me for the Sankey Diagram? Did you mean:

```public float VentilationGains => (float) Results.TotalVentilationHeatLosses.Where(x => x <= 0.0).Select(x => Math.Abs(x)).Sum()```

Reply:
Yes this is supposed to be a comment for you. Since you also process these lists for the outgoing arrows (ignoring negative values), I thought it consistent that you also process it for the ingoing arrows (ignoring positive values)

Reply:
Sorry I made a mistake. Give me a minute